content/en/docs/concepts/storage/volumes.md

```diff
@@ -1291,8 +1291,11 @@ Once a CSI compatible volume driver is deployed on a Kubernetes cluster, users
 may use the `csi` volume type to attach, mount, etc. the volumes exposed by the
```

Comment:
Should we link to the new ephemeral-volumes page somewhere at the top of this page? (and link to the persistent volumes page in the same area)

Reply:
The "background" section may need a more thorough rewrite. For example, it still says:
> A Kubernetes volume, on the other hand, has an explicit lifetime - the same as
the Pod that encloses it.

That became obsolete when persistent volumes were introduced, which was a while ago...

Let me punt on that for now for the sake of getting this PR merged before the docs deadline. I've created https://github.com/kubernetes/website/issues/22505 to track it.